app/src/main/java/com/amaze/filemanager/adapters/AppsAdapter.java

```diff
@@ -249,9 +242,23 @@ public void onPositive(MaterialDialog materialDialog) {
             });
 
             popupMenu.inflate(R.menu.app_options);
+            if(!isGooglePlayStoreAvailable()) {
+                popupMenu.getMenu().findItem(R.id.play).setVisible(false);
+            }
             popupMenu.show();
         });
 
     }
 
+    private final boolean isGooglePlayStoreAvailable() {
+        PackageManager pm = context.getPackageManager();
+        try {
+            pm.getPackageInfo(COM_ANDROID_VENDING, PackageManager.GET_ACTIVITIES);
+            return true;
+        } catch (PackageManager.NameNotFoundException ifNotInstalled) {
+            ifNotInstalled.printStackTrace();
```

Comment:
I think this just clutters the logcat, when printed it doesn't help, hence we shouldn't print it.